src/i18n/resources/ar.js

```diff
@@ -446,7 +446,7 @@ The FRA team
     nationalData: 'بيانات وطنية',
     analysisAndProcessing: 'تحليل البيانات الوطنية ومعالجتها',
     estimationAndForecasting: 'التوقعات والتنبؤات',
-    reclassification: 'إعادة التصنيف بما يتوافق وفئات تقييم الموارد الحرجية لعام 2020',
+    reclassification: 'إعادة التصنيف بما يتوافق وفئات تقييم الموارد الحرجية لعام {{cycleName}}',
```

Comment:
did you put cycleName at the beginning? here I see it at the end.
<img width="1361" alt="image" src="https://user-images.githubusercontent.com/1138156/212083873-b9b92044-3274-472f-a1ac-f9de5036163a.png">

Reply:
It should be rendered correctly on platform though (replaced the year with the variable)

localhost change
![image](https://user-images.githubusercontent.com/5508251/212102772-585760fa-ae27-492f-ae95-0b1a65cdb723.png)

production
![Screenshot 2023-01-12 at 16 07 23](https://user-images.githubusercontent.com/5508251/212102852-9246260b-57c6-4bdb-8f94-e5efe749a844.png)